projects/cashmere/src/lib/radio-button/radio-button.component.html

```diff
@@ -1,5 +1,7 @@
-<label class="hc-radio-container" [ngClass]="{'disabled': disabled}">
-    <input type="radio"
+<label class="hc-radio-container" [class.disabled]="disabled">
+    <input class="hc-radio-input"
+           [class.disabled]="disabled"
```

Comment:
I'm curious, do we also need to set [disabled]="disabled" here?

Reply:
That would improve accessibility. Good call.

Reply:
Oh, looks like it's already set. See line 9.

Reply:
So it is! :)